src/targets/gpu/kernels/include/migraphx/kernels/pooling.hpp

```diff
@@ -133,9 +133,8 @@ struct window
         auto win_start = generate_array<diff_int>(rank{}, [&](auto j) {
             diff_int w   = win[j];
             diff_int dim = i[j];
+	    (void) w;
```

Comment:
FYI, testing of JIT pooling in CI disabled. 

https://github.com/ROCm/AMDMIGraphX/issues/3234

Reply:
Thanks. This is already being verified by forcing it manually.

Reply:
> This is already being verified by forcing it manually -- in a separate build. 
Where ?

Reply:
Sorry, I meant in a private build. This is being tested by disabling this clause: https://github.com/ROCm/AMDMIGraphX/blob/b8c33b0754041cd61a0bd41396a1838b7c88a8c7/src/targets/gpu/lowering.cpp#L314, to turn on JIT kernels for pooling. So this will test all the pooling modes. `make check` and `test_verify` have been run successfully.